.pre-commit-config.yaml

```
@@ -0,0 +1,29 @@
+---
+repos:
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.1
+    hooks:
+      - id: remove-tabs
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: check-case-conflict
+      - id: detect-private-key
+      - id: check-ast
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
```

```suggestion
  - repo: https://github.com/psf/black
    rev: 19.10b0
    hooks:
      - id: black
        args: ['--line-length', '110'] 
```

black uses 88 characters as line length by default, for that reason is splitting lines shorter than 110
```  
-l, --line-length INTEGER       How many characters per line to allow.
                                  [default: 88]
```

---

fixed :+1: